app/src/main/java/org/fossasia/openevent/api/APIClient.java

```diff
@@ -37,6 +48,15 @@ public APIClient() {
                 .setConverter(new GsonConverter(gson))
                 .setEndpoint(Urls.BASE_GET_URL_ALT)
                 .setLogLevel(RestAdapter.LogLevel.FULL)
+                .setErrorHandler(new ErrorHandler() {
+                    @Override
+                    public Throwable handleError(RetrofitError cause) {
+                        ErrorDialogEvent errEvent = new ErrorDialogEvent(null, null);
+                        errEvent.ErrorHandlerFunction(cause, errEvent);
```

Comment:
I think we should drop the errorHandler in retrofit client, use the one from the process see [above](https://github.com/fossasia/open-event-android/pull/128#discussion_r48522034)